packages/synapse-client/src/util/fetchWithExponentialTimeout.ts

```diff
@@ -58,6 +61,26 @@ export const fetchWithExponentialTimeout = async <TResponse>(
     }
   }
 
+  return response
+}
+
+/**
+ * Fetches data, retrying if the HTTP status code indicates that it could be retried. Contains custom logic for
+ * handling errors returned by the Synapse backend.
+ * @throws SynapseClientError
+ */
+export const fetchWithExponentialTimeout = async <TResponse>(
```

Comment:
Refactor to use new method, behavior is the same to prevent breaking changes for other places where this function is used.